### ISSUE TITLE
Nested clickout components get triggered on click 

### ISSUE THREAD

**Issue by arempe93** [boblauer/react-onclickout#13]
If a clickout handler is nested within another, ie:

```jsx
<ClickOutHandler onClickOut={this.parentClickOut}>
    <ClickOutHandler onClickOut={this.childClickOut}>
    </ClickOutHandler>
</ClickOutHandler>
```

directly clicking within the nested component will trigger its onClickOut handler

Fiddle: http://jsfiddle.net/9f5edp9e/1

**Comment by boblauer** [comment]
Thanks for the fiddle, that helped a ton.

The fix has been published on npm, `@2.0.6`

**Comment by arempe93** [comment]
Wow that was quick! Thanks so much @boblauer
